Taskfile.yml

```diff
@@ -27,6 +33,9 @@ tasks:
     cmds:
       - python3 el.py -elp
       - dbt build --target prod
+  setup-bi:
```

Comment:
could we make this `bi-setup` and then instead of adding `dbt deps` to every build also add a `transform-setup` that wraps `dbt deps` instead? wdyt?

Reply:
Make total sense - clearly i was in 2 minds thanks for making it clear